### ISSUE TITLE
No icon for "Touch device" pseudo-mod

### ISSUE THREAD

**Issue by mcendu** [ppy/osu#6966]
**Describe the bug:** 
If player beat their personal highscore in a beatmap via a touch device in osu!stable, the placeholder icon (?) with the caption "Touch Device" appears instead of that shown on osu!web.

**osu!lazer version:** commit [`053e38c`](https://github.com/ppy/osu/commit/053e38c95fc97e60c4d3ca0098da929e22955a60) compiled with debug

**Comment by smoogipoo** [comment]
Yes, this is the same for many other mods. The icons haven't been all added to the game. I'm going to edit this issue to generalise it.

**Comment by smoogipoo** [comment]
@arflyte Is there any re-design of mod icons as of recent including the missing icons?

Missing:
```
General:
Day core

osu!mania:
1K to 9K
Dual stages
Mirror
Fade In (currently uses hidden icon)

osu!:
Traceable
```

**Comment by Tanza3D** [comment]
is there any plans for this? i'd be interested in trying to make a few if that's on the cards.

**Comment by peppy** [comment]
> is there any plans for this? i'd be interested in trying to make a few if that's on the cards.

Go for it!

**Comment by Joehuu** [comment]
Superseded by https://github.com/ppy/osu/issues/34248.
